html/pfappserver/lib/pfappserver/Form/Config/ProfileCommon.pm

```diff
@@ -254,6 +254,18 @@ has_field 'sms_pin_retry_limit' =>
              help => 'Maximum number of times a user can retry a SMS PIN before having to request another PIN. A value of 0 disables the limit.' },
 
   );
+=head2 login_attempt_retry_limit
+
+=cut
+
+has_field 'login_attempt_retry_limit' =>
+  (
+    type => 'PosInteger',
+    label => 'Login Attempt Retry Limit',
+    default => 0,
+    tags => { after_element => \&help,
+             help => 'Maximum number of times a user attempt to login before being blocked. A value of 0 disables the limit.' },
```

Comment:
Simplify this: `Limit the number of login attempts. A value of 0 disables the limit.`